pkgs/by-name/st/stash-clipboard/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "stash-clipboard";
+  version = "0.2.5";
+
+  src = fetchFromGitHub {
+    owner = "NotAShelf";
+    repo = "stash";
+    rev = "v${version}";
+    hash = "sha256-PEW28BocOhKylJXbFMSbAMJyrJW7JOeKmYVhlhrCQFc=";
+  };
+
+  cargoHash = "sha256-gEGuSe/WyxvwYic4BfnCFpRCnMN44n6DX3s0Aer4zBI=";
+
+  meta = with lib; {
+    description = "Wayland clipboard manager with fast persistent history and multi-media support";
+    homepage = "https://github.com/NotAShelf/stash";
+    license = licenses.mpl20;
+    maintainers = with maintainers; [
+      NotAShelf
+      fazzi
+    ];
+    mainProgram = "stash";
```

```suggestion
  meta = {
    description = "Wayland clipboard manager with fast persistent history and multi-media support";
    homepage = "https://github.com/NotAShelf/stash";
    license = lib.licenses.mpl20;
    maintainers = with lib.maintainers; [
      NotAShelf
      fazzi
    ];
    mainProgram = "stash";
```

Nested `with ...;` attributes are discouraged.